src/clsourcedependencies_parser.cc

```diff
@@ -63,7 +64,15 @@ bool CLSourceDependenciesParser::Parse(const string& content, string* err) {
       return false;
     }
 
+#ifdef _WIN32
+    string include_str;
+    if (!TryConvertUtf8ToAnsi(include_c_str, include_str)) {
```

Comment:
We should comment why we need this since it's not obvious. On Win32, Ninja std::strings are implicitly ANSI-encoded because they are used to call Win32 ANSI APIs. Without the conversion, extended characters of the ANSI encoding in paths would not be supported.